src/main.js

```diff
@@ -90,8 +93,39 @@ function compileTo(src, dest) {
     }
 }
 
+function compileAndRun(src) {
+    var js = compileSource(src);
+    var script = new vm.Script(js);
+    var root = global;
+    root.require = function(id) {
+        try {
+            return require(path.resolve(id));
+        } catch (Error) {
+            return require(id);
+        }
+    };
+    root.__filename = path.normalize(path.join(process.cwd(), src));
+    root.__dirname = path.dirname(root.__filename) + "/";
+
+    hook.hook(".sqg", function(src, filename) {
+        return compileSource(filename);
+    });
+
+    var ctx = vm.createContext(root);
+    script.runInContext(ctx);
+
+    hook.unhook(".sqg");
+}
+
+function compileTo(src, dest) {
+    var js = compileSource(src);
+    fs.writeFileSync(dest, js, UTF8);
+}
+
 if (argv.interactive) {
     repl.start();
+} else if (argv._.length === 1) {
+    compileAndRun(argv._[0]);
 } else if (argv._.length === 2) {
     compileTo(argv._[0], argv._[1]);
```

Comment:
Move the `compileTo` above the previous `else if` and check for `argv.output` and use that value.